MultiTarget/Test-Component-Support.ps1

```diff
@@ -0,0 +1,131 @@
+<#
+.SYNOPSIS
+    Tests WinUI support for a specific component and its requested MultiTargets.
+
+.DESCRIPTION
+    This script tests WinUI support for a specific component and its requested MultiTargets. It checks if the requested MultiTargets are supported by the component and if the WinUI version is compatible.
+
+.PARAMETER SupportedMultiTargets
+    Specifies known supported MultiTargets for the given component. Is retrieved if not specified.
+
+.PARAMETER RequestedMultiTargets
+    Specifies the MultiTarget TFM(s) to include for building the components. The default value is 'all'. If not specified, it will use the supported MultiTargets.
+
+.PARAMETER Component
+    Specifies the names of the components to build. Defaults to all components.
+
+.PARAMETER WinUIMajorVersion
+    Specifies the WinUI major version to use when building for Uno. Also decides the package id and dependency variant.
+
+.EXAMPLE
+    Build-Toolkit-Components -MultiTargets 'uwp', 'wasm' -DateForVersion '220101' -PreviewVersion 'local' -NupkgOutput 'C:\Output' -BinlogOutput 'C:\Logs' -EnableBinLogs -Components 'MyComponent1', 'MyComponent2' -ExcludeComponents 'MyComponent3' -Release -Verbose
```

Comment:
This example I think needs to be updated to match the new script parameters here?